client/state/sites/selectors/get-site-plan-slug.js

```diff
@@ -1,13 +1,8 @@
-/**
- * External dependencies
- */
-import { get } from 'lodash';
-
 /**
  * Internal dependencies
  */
 import getSitePlan from './get-site-plan';
 
 export default function getSitePlanSlug( state, siteId ) {
-	return get( getSitePlan( state, siteId ), 'product_slug' );
+	return getSitePlan( state, siteId )?.product_slug ?? null;
```

Comment:
Are we intentionally changing the default value from `undefined` to `null`?

Reply:
Yes, this was intentional -- it's common for Calypso selectors to return `null` if the result means "I don't know".

Reply:
👍  I wanted to make sure we've weighted any potential impact this has on the existing code that uses this selector.

Reply:
Now I see there are actually two `getSitePlanSlug` selector. The other one in `sites/plans/selectors` is slightly different, and returns `null` in the "not found" case 🙈